docs/products/kafka/kafka-connect/howto/enable-automatic-restart.rst

```diff
@@ -0,0 +1,30 @@
+Enable Apache Kafka Connect connectors auto restart on failures
+===============================================================
+
+In case of an Apache Kafka Connect connector failure, automatic task restart is generally not recommended. Proper investigation on the problem's cause should be performed before attempting to restart the failed connector to avoid experiencing similar problems in the future. However, sometimes a task can fail due to a rare problem like the dedicated node going out of memory due to a huge surge of data. 
+
+.. Note::
+
+    We observed cases when, using the Debezium source connector for PostgreSQL, the Apache Kafka connector stopped working after the PostgreSQL maintenance update, due to the PostgreSQL inability to create replication slots before failover. In such cases the connector automatic restart could be a valid solution to avoid the problem.
```

Comment:
"Debezium source connector for PostgreSQL, the Apache Kafka connector ", here is this Debezium source connector same as the Apache Kafka connector? If yes, I'd rephrase to avoid confusion to "We observed cases when the Debezium source connector for PostgreSQL stopped working because PostgreSQL was unable to create replication slots after the maintenance update of PostgreSQL. " Does it still have the same meaning?

Reply:
Thanks, fixed!